src/utils.js

```diff
@@ -107,6 +107,21 @@ export default class Utils {
 
     }
 
+    getAttr(attributeName, element) {
+        let result = null;
+        if(attributeName){
+            result = element.getAttribute(attributeName);
+        }
+
+        this.targetWindow.console.log(element);
```

Comment:
Are these `console.log()` for debug use?
If so, could you delete these lines?

Reply:
ah, yes, this is for debug use.

Reply:
removed